<?xml version="1.0"?>
<launch>

    <arg name="in_odom_topic" default="/odom/throttled"/>
    <arg name="in_cloud_topic" default="/cloud_synced"/>
    <arg name="out_cloud_topic" default="/cloud_map"/>
    <arg name="use_odom_as_correct_poses" default="true"/>

    <node pkg="graph_slam" name="create_map_node" type="create_map_node" output="screen">
        <param name="in_odom_topic" value="$(arg in_odom_topic)"/>
        <param name="in_cloud_topic" value="$(arg in_cloud_topic)"/>
        <param name="out_cloud_topic" value="$(arg out_cloud_topic)"/>
        <param name="use_odom_as_correct_poses" value="$(arg use_odom_as_correct_poses)"/>
    </node>

</launch>
